model/qti/container/ContainerGap.php

```diff
@@ -53,7 +51,8 @@ public function getValidElementTypes()
             'oat\\taoQtiItem\\model\\qti\\Tooltip',
             'oat\\taoQtiItem\\model\\qti\\feedback\\Feedback',
             \oat\taoQtiItem\model\qti\QtiObject::class,
-            'oat\\taoQtiItem\\model\\qti\\choice\\Gap'
+            'oat\\taoQtiItem\\model\\qti\\choice\\Gap',
+            \oat\taoQtiItem\model\qti\Figure::class,
```

Comment:
can we use `{classname}::class` in all of them?
please check the other classes that you touched as well